src/web_crawler/twir_crawler.rs

```diff
@@ -272,3 +294,15 @@ impl TwirCrawler {
         Ok(())
     }
 }
+
+fn get_progress_bar(text: &str) -> ProgressBar {
+    let progress_bar = ProgressBar::new(100);
```

Comment:
Sorry to be pedantic here, maybe it's no value - let me know what you think. We can add a "utils" module if you think it's a good idea. Put this function there and so we can define constants there to replace magic numbers. Something along the lines of const 

```rust
const PROGRESS_BAR_LENGTH: u64 = 100;
```

Reply:
Good point. A utils mod would be suitable here. Will create a utils mod and move it there.